src/openms/source/ANALYSIS/OPENSWATH/OpenSwathScoring.cpp

```diff
@@ -200,7 +200,8 @@ namespace OpenMS
     diascoring.dia_isotope_scores(transitions, spectrum, imrmfeature, scores.isotope_correlation, scores.isotope_overlap);
 
     // Peptide-specific scores
-    if (compound.isPeptide())
+    // useless for FFID
```

Comment:
hmm not ideal to refer back to tool that calls OpenSwathScoring. Is there a better comment (you can stille add a note (e.g., as done in FFId) )